bpf/bpf_lxc.c

```diff
@@ -262,6 +262,12 @@ static __always_inline int ipv6_l3_from_lxc(struct __ctx_buff *ctx,
 #ifdef ENABLE_NODEPORT
 		/* See comment in handle_ipv4_from_lxc(). */
 		if (ct_state.node_port) {
+#ifdef ENABLE_EGRESS_REVNAT_AT_HOST
+			/* Pass packet to next stack(Kernel) and let egress host perform
+			 * reverse NAT.
+			 */
+			return CTX_ACT_OK;
```

Comment:
In case there is BPF host routing enabled, we cannot send this up to the stack, but this needs to go to the phys netdev directly, see redirect_direct_v{4,6}().

Reply:
Does below flag is accurate to determine BPF host routing enabled ?
`ENABLE_REDIRECT_FAST`

Reply:
If NETFILTER_COMPAT_MODE is set, BPF host routing would be disabled. Hence, it shouldn't be a problem now.